app/controllers/concerns/curation_concerns/download_behavior.rb

```diff
@@ -13,10 +13,18 @@ def default_content_path
     # Render the 404 page if the file doesn't exist.
     # Otherwise renders the file.
     def show
-      if file
+      unless file
```

Comment:
Since you're already doing a type check on `file` in the `case` statement, can you move this into the `case` statement?

```ruby
case file
when ActiveFedora::File
#...
when nil
#...
else
#...
end
```